pkg/retention/space_default_retention_policy_service.go

```diff
@@ -0,0 +1,28 @@
+package retention
+
+import (
+	"github.com/OctopusDeploy/go-octopusdeploy/v2/pkg/newclient"
+	"github.com/OctopusDeploy/go-octopusdeploy/v2/pkg/services"
+)
+
+type SpaceDefaultRetentionPolicyService struct {
+	services.Service
+}
+
+const template = "/api/{spaceId}/retentionpolicies{/id}{?RetentionType}"
+
+func Get(client newclient.Client, spaceDefaultRetentionPolicyQuery SpaceDefaultRetentionPolicyQuery) (*SpaceDefaultRetentionPolicy, error) {
+	res, err := newclient.GetResourceByQuery[SpaceDefaultRetentionPolicy](client, template, spaceDefaultRetentionPolicyQuery)
```

Comment:
```suggestion
func Get(client newclient.Client, spaceDefaultRetentionPolicyQuery SpaceDefaultRetentionPolicyQuery) (*SpaceDefaultRetentionPolicyResource, error) {
	res, err := newclient.GetResourceByQuery[SpaceDefaultRetentionPolicyResource](client, template, spaceDefaultRetentionPolicyQuery)
```
I think this should return the resource so it has all the properties